app/assets/stylesheets/articles.scss

```diff
@@ -4,297 +4,74 @@
 @import 'variables';
 @import '_mixins';
 
+
 .home {
   position: relative;
-  min-height: 440px;
-  margin: auto;
+  min-height: 440px; // TODO: why?
```

Comment:
This might have been added back in the day when there was less content?

Reply:
Yea, I left some `// TODO` comments for myself to figure it out later, kind of notes :D 

Re this one: yes, Ben mentioned that too. He also mentioned it may not be needed anymore, but I'd rather clean it up in separate PR so I can test and be sure removing it won't do any harm.